SCons/Tool/ToolTests.py

```diff
@@ -39,6 +39,7 @@ def Detect(self, progs):
         return progs[0]
     def Append(self, **kw) -> None:
         self.dict.update(kw)
+    AppendUnique = Append  # wrong, but good enough for the use
```

Comment:
Maybe add a note here why this is being done?

Reply:
added,